spec/models/run_spec.rb

```diff
@@ -454,6 +475,32 @@
       end
     end
 
+    describe "#queue_for_portal" do
+      describe "when there are no answers" do
+        it "returns false and does not enqueue a job" do
+          expect(run.queue_for_portal).to be false
+        end
+      end
+
+      describe "when there are answers" do
+        let(:remote_endpoint) { nil }
+
+        it "returns the enqueued job when there is no remote_endpoint" do
+          run.open_response_answers << or_answer
+          expect(run.queue_for_portal).to be_instance_of(Delayed::Backend::ActiveRecord::Job)
+        end
+
+        describe "and there is a remote_enpoint" do
+          let(:remote_endpoint) { valid_remote_endpoint }
+
+          it "returns the enqueued job" do
+            stub_http_request(:post, remote_endpoint).to_return(:body   => "OK", :status => 200)
```

Comment:
If the delayed jobs are stopped, is it necessary to stub this?

Reply:
Delayed jobs are not stopped in this suite of tests - only for the suite that checks the dirty flag (as the job clears that flag before the test can check it).